docs/examples/opentelemetry-example-app/src/opentelemetry_example_app/grpc/route_guide_client.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://github.com/grpc/grpc/tree/master/examples/python/route_guide
+
+"""The Python implementation of the gRPC route guide client."""
+
+import logging
+import random
+
+import grpc
+
+from opentelemetry import trace
+from opentelemetry.ext.grpc import client_interceptor
+from opentelemetry.ext.grpc.grpcext import intercept_channel
```

Comment:
Nit: `ext.grpc.grpcext` seems repetitive here

Reply:
I didn't move this because it's part of the gRPC "extensions" from grpcio-opentracing. That is, the code in `ext.grpc.grpcext` could be gRPC itself, code in `ext.grpc` is specific to OT.

When we switch to gRPC's built-in interceptors from https://github.com/grpc/grpc/issues/8767 we can get rid of this package.